README.md

```diff
@@ -285,12 +318,26 @@ Now that you have built a Helm chart for your application, the process for deplo
 
 Deploy your Express.js application into Kubernetes using the following steps:
 
+0. ***microk8s only***: Push the image into the kubernetes container registry:
```

Comment:
minikube also has a registry addon, so this could be the same for both, and it would be needed with any cloud Kube